### PR TITLE
WEBEU2 - Redux Todo - James Eneh 🇳🇬

### PR DESCRIPTION
**Pull request by erozonachi**

### CODE REVIEW

File: todo/src/actions/action.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const CREATE_NEW_TODO = `CREATE_NEW_TODO`;
```
Comment (maximesalomon) [inline todo/src/actions/action.js:1]: You might want to rename the file to todoActions

File: todo/src/actions/creators.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { CREATE_NEW_TODO, TOGGLE_TODO_COMPLETED, DELETE_TODO } from './action';
```
Comment (maximesalomon) [inline todo/src/actions/creators.js:1]: You might want to rename the file to todoActionsCreators

File: todo/src/components/todos/AddTodoForm.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Form } from 'semantic-ui-react';
```
Comment (maximesalomon) [inline todo/src/components/todos/AddTodoForm.js:2]: 🔥

File: todo/src/components/todos/TodoList.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import LocalData from '../../stores/LocalData';
+
+const TodoList = props => {
+  const todoList = props.todos.map(todo => <TodoItem 
+    key={todo.id} 
+    todo={todo}
+    toggleCompleted={props.toggleCompleted}
+    deleteTodo={props.deleteTodo}  />);
+  
+  useEffect(() => {
```
Comment (maximesalomon) [inline todo/src/components/todos/TodoList.js:15]: Great use of the useEffect Hook 🎣

File: todo/src/reducers/reducer.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { CREATE_NEW_TODO, TOGGLE_TODO_COMPLETED, DELETE_TODO } from '../actions/action';
+import LocalData from '../stores/LocalData';
+
+const initialState = LocalData.fetchData('todoList') || [];
+
+export const todosReducer = (state = initialState, action) => {
```
Comment (maximesalomon) [inline todo/src/reducers/reducer.js:6]: I usually do single so --> todoReducer

File: todo/src/stores/store.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import { createStore } from 'redux';
+import { todosReducer as todos } from '../reducers/reducer';
```
Comment (maximesalomon) [inline todo/src/stores/store.js:2]: I understand what you are trying to do here but readability should trump shortness. Keep todoReducer*

* check previous comment

**Review by maximesalomon — COMMENTED** [review]
Hello James 👋 

Great job on this project 🔥

Amazing git commits as usual + Readme. Keep doing that!

I've added a couple of comments, go check them out

File: todo/src/actions/creators.js (hunk 2 above)
Comment (erozonachi) [reply todo/src/actions/creators.js:1]: Okay, got it 👍🏽

File: todo/src/stores/store.js (hunk 6 above)
Comment (erozonachi) [reply todo/src/stores/store.js:2]: Okay

**Comment by erozonachi** [comment]
Thank you! 👍🏽

**Pull request merged**